source/component/PasDoc_GenHtml.pas

```diff
@@ -1060,7 +1060,12 @@ procedure TGenericHTMLDocGenerator.WriteItemTableRow(
   { todo: assign a class }
   WriteStartOfTableCell('itemcode');
 
-  if MakeAnchor then WriteAnchor(Item.Name);
+  if MakeAnchor then begin
```

Comment:
From what I see, `TBaseItem.Signature` returns just `Name`. 

So this code should not need any special condition `    if Item is TPasRoutine then`, and can just do

```
if MakeAnchor then WriteAnchor(Item.Signature);
```